src/components/SearchBar/SearchBar.component.tsx

```diff
@@ -11,15 +12,21 @@ interface SearchBarProps {
     setSearchQuery: (query: string) => void;
 }
 
-function SearchBar({ setFilter, setSort, setSearchQuery }: SearchBarProps) {
-    const [selectedFilters, setSelectedFilters] = useState<string[]>([]);
+const categories: Readonly<Pick<Category, 'id' | 'name'>[]> = [
+    { id: 1, name: 'Clothes' },
+    { id: 2, name: 'Electronics' },
+    { id: 3, name: 'Furniture' },
+    { id: 4, name: 'Shoes' },
+    { id: 5, name: 'Miscellaneous' },
+];
+
+export function SearchBar({ setFilter, setSort, setSearchQuery }: SearchBarProps) {
```

Comment:
<img width="494" alt="Screenshot 2024-07-06 at 18 49 57" src="https://github.com/VasylBiliak/fe-react-2024/assets/115080771/c36de7db-9c75-4d24-bdd5-b3c837e917a7">

need rewrite search bar according design for mobile version